epic/check_autobranch.go

```diff
@@ -130,6 +133,18 @@ func isRelatedToAutoBranchBodyWithCheckSuiteEvent(ev *github.CheckSuiteEvent) fu
 	}
 }
 
+func inProgressWithStatusEvent(ev *github.StatusEvent) func() bool {
+	return func() bool {
+		return *ev.State == "pending"
+	}
+}
+
+func inProgressWithCheckSuiteEvent(ev *github.CheckSuiteEvent) func() bool {
+	return func() bool {
+		return *ev.CheckSuite.Status != "completed" || *ev.CheckSuite.Conclusion == "neutral"
```

Comment:
@yamachu 

[As your comment](https://github.com/voyagegroup/popuko/pull/278#issue-1053455717), I feel `*ev.CheckSuite.Conclusion == "neutral"` does not mean _in progress_ state directly.

But this code will check whether `*ev.CheckSuite.Conclusion` is `neutral` after if `*ev.CheckSuite.Status != "completed"` equals to that `*ev.CheckSuite.Status` indicates some stopped states. 

I concern that this function will return `true` in almost case. Is this right?

Reply:
> I concern that this function will return true in almost case. Is this right?

Yes, it enforces the order of use, which doesn't seem to be a good function...
Rather, what should be expressed is not whether it is in progress or not, but whether the state is determined or not.